### PR TITLE
Add config files

### PR DESCRIPTION
**Pull request by nickbristow**
Added dotenv for configuring hard coded URLs

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -8,6 +8,7 @@ gem 'http'
 gem 'activesupport'
 gem 'json-jwt'
 gem 'jwt'
+gem 'dotenv-rails'
```
Comment (zachmargolis) [inline Gemfile]: since this is sinatra, not rails, is there a plain old `dotenv` gem we should use instead?

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1,7 @@
+# See http://help.github.com/ignore-files/ for more about ignoring files.
+#
+# If you find yourself ignoring temporary files generated by your text editor
+# or operating system, you probably want to add a global ignore instead:
+#   git config --global core.excludesfile '~/.gitignore_global'
+
+.env
```
Comment (zachmargolis) [inline .gitignore]: missing a newline at the end of the file!

File: README.md [hunk 3]
```diff
@@ -6,13 +6,15 @@ An example of a Relying Party for OpenID Connect written as a simple Sinatra app
 
 These instructions assume [`identity-idp`](https://github.com/18F/identity-idp) is also running locally at `http://localhost:3000`. This sample sp is configured to run on `http://localhost:9292`.
 
-1. Set up the environment with:
+1. Copy `.env.example` to a new file `.env` and set variables accordingly
```
Comment (zachmargolis) [inline README.md]: Can we make that part of `make setup` ?

File: .env.example [hunk 4]
```diff
@@ -0,0 +1,17 @@
+IDP_SP_URL: 'http://localhost:3000/api/service_provider'
+ACR_VALUES: 'http://idmanagement.gov/ns/assurance/loa/1'
+REDIRECT_URI: 'http://localhost:9292/auth/result'
+
+# IDP_SSO_URL: 'http://localhost:3000/api/saml/auth'
+# DASHBOARD_SECRET_KEY_BASE: 'changeme'
+# IDP_SLO_URL: 'http://localhost:3000/saml/logout'
+# LOGIN_DASHBOARD_TOKEN: 'sekret'
+# NEW_RELIC_LICENSE_KEY: 'changeme'
```
Comment (zachmargolis) [inline .env.example]: newrelic is not configured yet, should we just remove this? basically all of them except `SP_NAME` and `SP_PASS`

File: .env.example [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
+# SMTP_ADDRESS: 'changeme'
+# SMTP_DOMAIN: 'changeme'
+# SMTP_PASSWORD: 'changeme'
+# SMTP_USERNAME: 'changeme'
+# SP_NAME: 'changeme'
+# SP_NAME: 'changeme'
+# SP_PASS: 'changeme'
```
Comment (zachmargolis) [inline .env.example]: duplicate line?

File: .gitignore (hunk 2 above)
Comment (nickbristow) [inline .gitignore]: should be there now

**Comment by nickbristow** [comment]
@zachmargolis should be good now!?

**Pull request merged**